### PR TITLE
Increase frequency of gene strains for botany infusions

### PR DESCRIPTION
**Pull request by TDHooligan**
## About the PR 

This adds a variable to the standard HYPCommut and HYPMutation procs in plants that multiplies the odds of gene strains appearing, using it when mutagenic reagents are infused in the PlantMaster.

This means there are no increased odds of these mutations happening during regular plant-based botany, but infusing mutagenics will be significantly less tedious.

This patch does not affect the odds of reagents causing mutations, which have pretty reasonable odds.

The initial patch sets the odds of getting a gene strain from infusion to:
Radium, Uranium, Stable Mutagen: 20%
Unstable Mutagen: 30%

## Why's this needed? 

Nerdy botanists are at the whims of some pretty miserable RNG (~3.33-5% odds for a given desired gene strain for one seed) and can be plastered to a screen for most of a round trying to splice for fancy gene strain combos. This should improve the odds that higher level botanists can make fancy plants and have some fun with them before the shuttle is called.

## Changelog

```changelog
(u)TDHooligan
(+)Increased frequency of gene strains when infusing with the PlantMaster.
```

### CODE REVIEW

**Comment by ZeWaka** [comment]
`severity` is a bit confusing imo

**Comment by TDHooligan** [comment]
changed it over to something more descriptive. any fuss with the balance change?

File: code/obj/machinery/plantpot.dm [hunk 1]
```diff
@@ -1647,7 +1646,7 @@ proc/HYPgeneticanalysis(var/mob/user as mob,var/obj/scanned,var/datum/plant/P,va
 	boutput(user, message)
 	return
 
-proc/HYPnewmutationcheck(var/datum/plant/P,var/datum/plantgenes/DNA,var/obj/machinery/plantpot/PP)
+proc/HYPnewmutationcheck(var/datum/plant/P,var/datum/plantgenes/DNA,var/obj/machinery/plantpot/PP, var/frequencymult = 1)
```
Comment (Tarmunora) [inline code/obj/machinery/plantpot.dm:1649]: never called with a non-1 mult?

Comment (TDHooligan) [reply code/obj/machinery/plantpot.dm:1649]: leftover from adding similar changes to mutation odds that i reverted, though I didn't take it out due to the similarity of the two functions. it's fine reverted or not, though i think it's reasonably likely to be used if anyone actually pokes at botany any time soon.

**Pull request merged**